.circleci/config.yml

```diff
@@ -1,156 +1,48 @@
 # Setup in CircleCI account the following ENV variables:
 # IS_PRODUCTION (default: 0)
-# IS_ENTERPRISE (default: 0)
 # PACKAGECLOUD_ORGANIZATION (default: stackstorm)
 # PACKAGECLOUD_TOKEN
```

Comment:
These lines could be removed as well.

Previously used for deploying the deb/rpm to PackageCloud, obsolete now with the packages removal.